src/diff/children.js

```diff
@@ -391,6 +391,14 @@ export function toChildArray(children, out) {
 	return out;
 }
 
+/**
+ * @param {VNode} node
+ * @returns {boolean}
+ */
+function nodeMatches(node) {
```

Comment:
Adding additional functions like this is most likely either going ot be inlined or when not inlined make the bundle bigger